apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServerCommon.kt

```diff
@@ -68,18 +71,64 @@ suspend fun writeResponse(sink: BufferedSink, mockResponse: MockResponse, versio
 class MockResponse(
     val statusCode: Int = 200,
     val body: ByteString = ByteString.EMPTY,
-    val chunks: List<MockChunk> = emptyList(),
+    chunks: List<MockChunk> = emptyList(),
     val headers: Map<String, String> = emptyMap(),
     val delayMillis: Long = 0,
+    private val waitForMoreChunks: Boolean = false,
 ) {
+  var chunks: List<MockChunk> = chunks
```

Comment:
In addition to the channel, still keep the List, as it is useful to do asserts in tests